roles/awx/tasks/main.yml

```diff
@@ -9,28 +9,6 @@
       tags:
         - awx
 
-# Normally this PVC is created by the awx-postgres statefulset, but this prevents resizing so we create it beforehand
```

Comment:
If possible the AWX Postgres should be configured to use this existing PVC

Reply:
Thanks! Should get evaluated in https://ticketsystem.dbildungscloud.de/browse/OPS-5484